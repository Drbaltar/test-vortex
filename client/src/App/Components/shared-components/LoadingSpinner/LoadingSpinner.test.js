import React from 'react';
import { shallow } from 'enzyme';
import LoadingSpinner from './LoadingSpinner';

describe('LoadingSpinner', () => {
    const wrapper = shallow(<LoadingSpinner />);

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('renders the spinner border component', () => {
        expect(wrapper.find('.spinner-border').exists()).toBe(true);
    });
});