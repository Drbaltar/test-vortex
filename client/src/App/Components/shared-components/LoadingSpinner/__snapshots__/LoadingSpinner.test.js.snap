// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner renders correctly 1`] = `
<div
  className="d-flex justify-content-center mb-3"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
